spec/reek/smells/attribute_spec.rb

```diff
@@ -127,5 +127,15 @@ class OtherKlass
       '
       expect(src).to reek_of(:Attribute)
     end
+
+    it 'can be suppressed with a preceding code comment' do
```

Comment:
Hm. That would basically repeat the test on line 31. Maybe I should move this test up there?